sdk/packages/model-instance-client/src/client.ts

```diff
@@ -28,8 +28,8 @@ import type { DocumentState, UnknownContent } from './types.js'
 export type CreateSingletonParams = {
   /** The model's stream ID */
   model: StreamID
-  /** The controller of the stream (DID string or literal string) */
-  controller: DIDString | string
```

Comment:
this is losing the ability for the user to set the controller DID string explicitly, but maybe that's fine?  Almost everyone is going to be using did-session anyway so maybe that doesn't really matter as a feature

Reply:
Yeah, I think that should be ok.

I _could_ have left the DID strings in but that would have meant leaving it to callers to check whether the DID had a parent PKH and then to pass that in, which felt risky.